charts/rancher-eks-operator/1.0.1000/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-eks-operator
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/eks-operator
+sources:
+  - "https://github.com/rancher/eks-operator"
+version: 1.0.1000
```

Comment:
I am confused here:
this is a new version that does not exist in the released-v2.5 branch https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-eks-operator
How could it show up as an addition in this PR? 

Is my understanding right that your PR is to pull the release-v2.5 into the dev-v2.5 branch?

Reply:
The same thing is happening for gke-operator 1.1.1

This is my understanding:
The files are not new, but the commits that contain the files are new. This is the same thing that was happening in the `dev-2.5` to `release-v2.5` PR, which led to this rebase. After this rebase, the release PR should have a cleaner history.

Reply:
@thedadams  the above dev-to-release mock PR looks good.